changes/6825.fixed

```diff
@@ -0,0 +1 @@
+Fixed missing Linkify for manufacturer on platform table.
```

Comment:
Added links to the `manufacturer` column of the Platform table.

Reply:
Should probably change the name of the changelog fragment from `.fixed` to `added`.